### DISCUSSION TITLE
Using this docker for main internet gateway

### DISCUSSION THREAD

**Discussion by chrispazz** [bubuntux/nordvpn#190] [Q&A]
Hello,
currently I am using a pfsense firewall with installed the openvpn client to connect to nordvpn (UDP mode) and I am routing all traffic thru this.

Is it possibile to run this docker with Nordlynx protocol and have pfsense routing all traffic thru this Nordlynx gateway?

PS my pfsense is a virtualized machine on my synology nas where I run docker too.

**Comment by Groofster** [comment]
Technically it should be possible but I would absolutely advice you NOT to go there. You will create a lot of dependancies in your network to get an internet connection.

Simple example: you update this docker image and it bugs out for whatever reason. Now you have to reroute your network only to get the last (working) image from the internet.
